packages/calcite-components/src/components/block-group/block-group.e2e.ts

```diff
@@ -0,0 +1,550 @@
+  <div slot=${BLOCK_SLOTS.icon}>✅</div>
+  <div>content</div>
+  <label slot=${BLOCK_SLOTS.control}>test <input placeholder="control" /></label>
+</calcite-block>`;
+
+describe("calcite-block-group", () => {
+  describe("defaults", () => {
+    defaults("calcite-block-group", [
+      {
+        propertyName: "disabled",
+        defaultValue: false,
+      },
+      {
+        propertyName: "dragEnabled",
+        defaultValue: false,
+      },
+      {
+        propertyName: "group",
+        defaultValue: undefined,
+      },
+      {
+        propertyName: "label",
+        defaultValue: undefined,
+      },
+      {
+        propertyName: "loading",
+        defaultValue: false,
+      },
+    ]);
+  });
+
+  describe("reflects", () => {
+    reflects("calcite-block-group", [
+      {
+        propertyName: "disabled",
+        value: true,
+      },
+      {
+        propertyName: "dragEnabled",
+        value: true,
+      },
+      {
+        propertyName: "group",
+        value: "test",
+      },
+      {
+        propertyName: "loading",
+        value: true,
+      },
+    ]);
+  });
+
+  describe("renders", () => {
+    renders("calcite-block-group", { display: "block" });
+  });
+
+  describe("is focusable", () => {
+    focusable(html`<calcite-block-group> ${blockHTML} </calcite-block-group>`, {
+      focusTargetSelector: "calcite-block",
+    });
+  });
+
+  describe("honors hidden attribute", () => {
+    hidden("calcite-block-group");
+  });
+
+  describe("accessible", () => {
+    accessible(html`<calcite-block-group> ${blockHTML} </calcite-block-group>`);
+  });
+
+  describe("disabled", () => {
+    disabled(html`<calcite-block-group> ${blockHTML} </calcite-block-group>`, { focusTarget: "child" });
+  });
+
+  it("should set the dragHandle property on items", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      html`<calcite-block-group id="root" drag-enabled group="my-block-group">
+        <calcite-block id="one" heading="one" label="One"></calcite-block>
+        <calcite-block id="two" heading="two" label="Two"></calcite-block>
+        <calcite-block id="three" heading="three" label="Three"></calcite-block>
+      </calcite-block-group>`,
+    );
+
+    await page.waitForChanges();
+    await page.waitForTimeout(DEBOUNCE.nextTick);
+
+    const items = await page.findAll("calcite-block");
+
+    for (let i = 0; i < items.length; i++) {
+      expect(await items[i].getProperty("dragHandle")).toBe(true);
+    }
+
+    const root = await page.find("#root");
+
+    root.setProperty("dragEnabled", false);
+    await page.waitForChanges();
+    await page.waitForTimeout(DEBOUNCE.nextTick);
+
+    for (let i = 0; i < items.length; i++) {
+      expect(await items[i].getProperty("dragHandle")).toBe(false);
+    }
+  });
+
+  it("should set the dragHandle property on items which are not direct children", async () => {
```

Comment:
This seems unsupported since `block-group`'s default slot expects block as children. This might affect the `observe` props passed to the mutation observer.

Reply:
Yeah this was copied from list, I can remove this test.